packages/daemon/src/services/index.ts

```diff
@@ -376,6 +429,15 @@ export const handleVertexAccepted = async (context: Context, _event: Event) => {
     await mysql.rollback();
     logger.error(e);
 
+    if (e instanceof Error) {
+      logger.error('Error handling vertex accepted', {
+        error: e.message,
+        stack: e.stack,
+      });
+    } else {
+      logger.error('Error handling vertex accepted', { error: e });
+    }
+
```

Comment:
Should we remove this first `logger.error(e);`?

Reply:
Yes, thanks